### DISCUSSION TITLE
Make it easy to clone a repo on mobile by exposing the repo’s url, so it can be easily copy/pasted.

### DISCUSSION THREAD

**Discussion by weedmapsmike** [community/community#30151] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.73.0 (72950773)
Device: iPad Air 5 (iOS 15.6.1)
</details>

**Comment by taki-on** [comment]
You can get the repository URL from the repository screen's Share function. You can tap on the `...` button on the top right, and you will find a share button, where you can copy the repository URL. Does this help you?
